docs/source/using/troubleshooting.adoc

```diff
@@ -314,3 +314,28 @@ You can configure the URL by running:
 $ minishift config set check-network-http-host <URL>
 ----
 
+[[github-api-rate-limit-exceeded]]
+== GitHub API rate limit exceeded
+
+When you run `minishift start` or `minishift update`, it might make a request to GitHub to get the latest tarball of the Minishift or OpenShift binary.
+Sometimes, during these requests, you might receive a 403 forbidden status from GitHub if your network reached the API rate limit and the command does not succeed, for example:
```

Comment:
"your IP address reached the API rate limit" - an IP address cannot reach a limit. It is the limit for a given IP which gets exceeded.